doc/api/child_process.md

```diff
@@ -1133,6 +1133,10 @@ may not actually terminate the process.
 
 See kill(2) for reference.
 
+On Windows the child process will be forcefully terminated since
+`SIGKILL` and `SIGTERM` signals are not available.
```

Comment:
Should this say `SIGINT` and `SIGTERM` instead? 
If I'm not mistaken, `SIGINT` isn't available here either.
And `SIGKILL`, if available, forcibly terminates (so it's unavailability doesn't help explain why child process must be forcefully terminated.. `SIGKILL` wouldn't be useful for avoiding forceful termination anyways).